packages/remix-server-runtime/server.ts

```diff
@@ -95,10 +95,10 @@ export const createRequestHandler: CreateRequestHandlerFunction = (
 
     let matches = matchServerRoutes(routes, url.pathname);
     let handleError = (error: unknown) => {
-      let devServerHooks = getDevServerHooks();
-      if (devServerHooks && error instanceof Error) {
-        devServerHooks.fixStacktrace(error);
+      if (mode === ServerMode.Development) {
```

Comment:
We want to make sure these hooks are never called outside of dev, especially since the hooks are coming from a global that can be modified by anyone.